package.json

```diff
@@ -6,7 +6,8 @@
   "type": "module",
   "scripts": {
     "format": "npx prettier --write .",
-    "lint": "npx eslint --fix ."
+    "lint": "npx eslint --fix .",
+    "codespace-google-auth": "printenv GOOGLE_CREDENTIALS > service_account_key.json && export GOOGLE_APPLICATION_CREDENTIALS=service_account_key.json"
```

Comment:
<img width="884" alt="Screenshot 2023-06-23 at 10 55 49 PM" src="https://github.com/utdata/icj-project-rig/assets/5885605/882ea8db-a7b0-4a15-ada4-075ddd02b98a">

Here is it being tested on a Codespace.